scene/3d/skeleton_3d.cpp

```diff
@@ -491,7 +491,7 @@ void Skeleton3D::set_bone_name(int p_bone, const String &p_name) {
 
 	for (int i = 0; i < bone_size; i++) {
 		if (i != p_bone) {
-			ERR_FAIL_COND(bones[i].name == p_name);
+			ERR_FAIL_COND_MSG(bones[i].name == p_name, "Bone \"" + p_name + "\" already exists in Skeleton3D.");
```

Comment:
Previously you used "sheleton", now "Skeleton3D". It would probably be better to use the same term.